src/interfaces/Cafeteria.ts

```diff
@@ -3,12 +3,16 @@ import { CAFETERIA_CATEGORY, CAFETERIA_TIME } from 'static/cafeteria';
 export type CafeteriaTime = typeof CAFETERIA_TIME[number]['type'];
 export type CafeteriaCategory = typeof CAFETERIA_CATEGORY[number]['placeName'];
 export type CafeteriaMenu = {
+  created_at: string;
   id: number;
   date: string;
+  image_url:string | null;
   type: CafeteriaTime;
   place: CafeteriaCategory;
   price_card: number;
   price_cash: number;
   kcal: number;
   menu: string[];
+  sold_out: boolean;
+  updated_at: string;
```

Comment:
이번에 API Response구조가 변경되면서 `sold_out`도 변경됐어요!
또한 변경됨을 표시하기 위해 `changed_at`도 필요해보입니당
```suggestion
  soldout_at: string;
  updated_at: string;
  changed_at: string;
```

Reply:
안그래도 변경됨을 표시할 조건이 updated_at밖에 없나 생각했는데 필드에 수정이 있었군요!
바로 반영해서 수정하겠습니다~